src/components/cards/ArnsCard/ArnsCard.tsx

```diff
@@ -0,0 +1,70 @@
+import './styles.css';
+import { ArnsCardProps } from '../../../types';
+import { useEffect, useState } from 'react';
+import { useStateValue } from '../../../state/state';
+import { ArnsDefault } from '../../icons';
+import axios from 'axios';
+
+function ArnsCard({ name }: ArnsCardProps) {
+  const [{ arnsSourceContract }] = useStateValue();
+  const [antDetails, setAntDetails] = useState({
+    name: 'arns',
+    gateway: 'arweave.net',
```

Comment:
hmm, thinking we should pull the global `gateway` for this

Reply:
figured we can wait until wallet info is setup - it can be pulled from the global arweave instance. We can add it now if you like.